easybuild/easyconfigs/n/numactl/numactl-2.0.11-iccifort-2016.3.210-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'numactl'
+version = '2.0.11'
+
+checksums = ['d3bc88b7ddb9f06d60898f4816ae9127']
+
+homepage = 'http://oss.sgi.com/projects/libnuma/'
+description = """The numactl program allows you to run your application program on specific cpu's and memory nodes.
+ It does this by supplying a NUMA memory policy to the operating system before running your program.
+ The libnuma library provides convenient ways for you to add NUMA memory policies into your own program."""
+
+toolchain = {'name': 'iccifort', 'version': '2016.3.210-GCC-5.4.0-2.26'}
+
+source_urls = ['ftp://oss.sgi.com/www/projects/libnuma/download/']
+sources = [SOURCE_TAR_GZ]
+
+# builddependencies = [('Autotools', '20150215', '', ('GCC', '5.4.0-2.26'))]
+
+# preconfigopts = './autogen.sh && '
```

Comment:
@jhein32 actually, you can just drop these two lines, they indeed don't seem to be needed for this version of `numactl`; the other easyconfigs for version 2.0.11 don't have these lines either